swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -35,6 +37,14 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
 
     super.init()
 
+    // Log version information immediately on startup
```

Comment:
I don't mind adding these logs but I also wouldn't be opposed to leaving them out of this PR.

Reply:
I'll keep them here if you don't mind